<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formatCovertWidget</class>
 <widget class="QWidget" name="formatCovertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>219</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>219</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>formatCovert</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(200,200,200);
}

</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color: rgb(60,60,60,0);
	color: #000;
	font: 9pt &quot;等线l&quot;;
	border:2px solid rgb(0,0,0);
	border-radius:10px;
	padding-left:5px;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>90</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
	background-color: rgb(200,200,200);
	border: 2px solid rgb(0,0,0);
	border-left:0px solid #fff;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
	color: rgb(0,0,0);
	font: 75 9pt &quot;等线&quot;;
}

QComboBox::drop-down
{
	width:0px;
	border-style: none;
	background-color:rgb(0,0,0,0);
}</string>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>.mp4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.avi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.flv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.mkv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.rmvb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.mp3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255,0);
	border: 2px solid rgb(0,0,0);
	border-right:0px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
	color: rgb(0,0,0);
	font: 75 9pt &quot;等线&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="convertButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255,0);
	border: 2px solid rgb(0,0,0);
	border-radius: 20px;
	color: rgb(0,0,0);
	font: 75 11pt &quot;等线&quot;;
}
QWidget:hover
{
	font-size:12pt;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
QWidget:pressed 
{
	font-size:9pt;
}</string>
   </property>
   <property name="text">
    <string>convert</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255,0);
	border: 2px solid rgb(0,0,0);
	border-radius: 10px;
	color: rgb(0,0,0);
	font: 75 9pt &quot;等线&quot;;
}
QWidget:hover
{
	font-size:10pt;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
QWidget:pressed 
{
	font-size:8pt;
}</string>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
